### PR TITLE
Better errors when Azure tags are missing

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

**Comment by djmitche** [comment]
Oh, I'll move that to loadTaggedData, plz hold

**Comment by djmitche** [comment]
ok, fixed :)

File: changelog/T7C5hp72RW64BbyOnIx0yQ.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+audience: worker-deployers
+level: patch
+---
+Worker-runner now gives better error messages when it does not information such as the RootURL in its tagged data.
```
Comment (milescrabill) [inline changelog/T7C5hp72RW64BbyOnIx0yQ.md:4]:
```suggestion
Worker-runner now gives better error messages when it does not have information such as the RootURL in its tagged data.
```

Comment (djmitche) [reply changelog/T7C5hp72RW64BbyOnIx0yQ.md:4]: lol words

**Pull request merged**